components/SourceTag.vue

```diff
@@ -1,26 +1,56 @@
 <template>
   <span
-    class="tag-element"
+    :class="['tag-element', 'font-sans', 'font-medium', 'ml-2']"
     :title="title"
-    :style="{ backgroundColor: background, color: textColor, border: border }"
+    :style="{
+      backgroundColor: text ? computedColor(text, 80, 95) : background,
+      color: text ? computedColor(text, 50, 30) : textColor,
+      // borderColor: text ? computedColor(text, 80, 90) : border,
+    }"
   >
     {{ text }}
   </span>
 </template>
 
 <script>
+import colors from 'tailwindcss/colors';
+
 export default {
   props: {
     text: String,
     title: String,
     textColor: {
-      default: '#ffffff',
+      type: String,
+      default: colors.slate[900],
     },
     background: {
-      default: '#767676',
+      type: String,
+      default: colors.slate[100],
     },
     border: {
-      default: '1px solid #999999',
+      type: String,
```

Comment:
This is a bit of a silly one, but it generates a random color for each source. I thought it was cute. :D